projects/cashmere/src/lib/banner/hc-banner.scss

```diff
@@ -1,110 +1,57 @@
-@import '../sass/colors';
-@import '../sass/functions.scss';
-@import '../sass/mixins.scss';
+@import '../sass/banner.scss';
 
 .hc-banner {
-    display: flex;
-    align-items: center;
-    justify-content: flex-start;
-    width: 100%;
-    padding: 10px;
-    z-index: $zindex-subnav;
+    @include hc-banner();
+}
 
-    &.fixed-top {
```

Comment:
This `fixed-top` class isn't referenced anywhere in Cashmere (documentation, examples, etc.). It appears to be orphaned. I deleted it, but I can add it back if we believe it's in use.